### PR TITLE
Feat: Add git branches to Algernon

### PR DESCRIPTION
**Pull request by leonardhorns**

### CODE REVIEW

**Comment by yannik-dittmar** [comment]
PR is still a draft, not reviewing. Also the test coverage seems to have decreased quite a bit. Have you written test for your stuff yet?

**Comment by leonardhorns** [comment]
Tests are done, but are currently still marked as expected failures until `SquitBrowser` is added to the CI on this branch. That should resolve the coverage issue.
Review can wait until marked ready for review...

File: packages/AlgernonTests.package/ALGLinting.class/instance/collectTestObjects.st [hunk 1]
```diff
@@ -4,5 +4,10 @@ collectTestObjects
...
 	classes := classes select: [ :each | each class category beginsWith: 'Algernon' ].
 	(Smalltalk classNamed:	#SquitBrowser) ifNil:
-		[classes := classes difference: {ALGGitProjectType . ALGTestGitProjectType}].
+		[classes := classes difference:
+			{ALGGitProjectType.
+			  ALGTestGitTypes.
+			  ALGGitBranchType.
+			  ALGGitLocalBranchType.
+			  ALGGitRemoteBranchType}].
 	^ classes collect: [ :each | each as: SLTestObject ].
```
Comment (gwauge) [inline packages/AlgernonTests.package/ALGLinting.class/instance/collectTestObjects.st:13]: Punkt am Ende weg

File: packages/AlgernonTests.package/ALGTestGitTypes.class/instance/setUpGitStamp.st [hunk 2]
```diff
@@ -0,0 +1,10 @@
+helpers
+setUpGitStamp
+
+	self
+		previousGitStampDefaultName: ([GitStamp defaultName] on: GitStampNameRequest do: ['']);
+		previousGitStampDefaultEmail: ([GitStamp defaultEmail] on: GitStampEmailRequest do: ['']).
+	
+	GitStamp
+		defaultName: 'Hasso';
```
Comment (gwauge) [inline packages/AlgernonTests.package/ALGTestGitTypes.class/instance/setUpGitStamp.st:9]: ❤️

**Review by gwauge — APPROVED** [review]
Sau starke Arbeit Leo!

Hab nur den einen Punkt gesehen wegen coding standards aber den kannst du eigentlich auch lassen sonst und wir refactoren ihn dann auf dem dev.

**Pull request merged**